github/ci/prow-deploy/files/jobs/kubevirt/bridge-marker/bridge-marker-presubmits-0.6.yaml

```diff
@@ -0,0 +1,33 @@
+---
+presubmits:
+  kubevirt/bridge-marker:
+    - name: pull-bridge-marker-test
```

Comment:
maybe need to add release-0.6 to the name ?

Reply:
Sure, done